### PR TITLE
Feat/textarea colors 2020

### PR DESCRIPTION
**Pull request by danethurber**

### CODE REVIEW

File: packages/textarea/src/css/index.js [hunk 1]
```diff
@@ -17,27 +24,27 @@ export default {
...
 
-    '&:focus': { outline: 'none' }
+    '&:focus': { outline: 'none' },
+    '&::placeholder': { color: colorsTextIcon.lowOnLight }
   },
   [`.psds-text-area__field.psds-theme--${themeNames.light}`]: {
     '&:focus': { border: '1px solid transparent' }
   },
 
+  [`.psds-text-area__field.psds-theme--${themeNames.light}`]: {
```
Comment (jaketrent) [inline packages/textarea/src/css/index.js]: This selector is a dup and can be combined with the one above.

File: packages/textarea/src/css/index.js [hunk 2]
```diff
@@ -17,27 +24,25 @@ export default {
...
   [`.psds-text-area__field.psds-theme--${themeNames.light}`]: {
+    border: `1px solid ${colorsBorder.highOnLight}`,
     '&:focus': { border: '1px solid transparent' }
   },
 
   [`.psds-text-area__field--error.psds-theme--${themeNames.light}`]: {
     border: '1px solid transparent'
   },
-  [`.psds-text-area__field.psds-theme--${themeNames.light}`]: {
-    background: colors.white,
```
Comment (jaketrent) [inline packages/textarea/src/css/index.js]: @danethurber Did you still want background: white?  I'm not seeing this transferred in the refactor.

Comment (danethurber) [reply packages/textarea/src/css/index.js]: uh oh....i'll fix

Comment (danethurber) [reply packages/textarea/src/css/index.js]: looks like i added a white background to the field regardless of theme. so this change is correct

**Pull request merged**